dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/ResourcePolicyRestRepository.java

```diff
@@ -52,9 +95,211 @@ public ResourcePolicyRest findOne(Context context, Integer id) {
         throw new RepositoryMethodNotImplementedException(ResourcePolicyRest.NAME, "findAll");
     }
 
-
     @Override
     public Class<ResourcePolicyRest> getDomainClass() {
         return ResourcePolicyRest.class;
     }
-}
+
+    @PreAuthorize("hasPermission(#resourceUuid, 'dspaceObject', 'ADMIN')")
+    @SearchRestMethod(name = "resource")
+    public Page<ResourcePolicyRest> resource(@Parameter(value = "uuid", required = true) UUID resourceUuid,
```

Comment:
Could you add JavaDocs to this new method?  This also looks like it should be named something like `findByResource` as it's returning all ResourcePolicies that match a given Resource.